layouts/index.html

```diff
@@ -0,0 +1,36 @@
+{{ define "main" }}
+<div class="featured-story featured-story-nid-default-featured-story featured-story-light" id="featured-story" data-publish-target="asciidoc">
```

Comment:
We should remove the data-publish target at least, if not the ID as well to stop things from colliding if possible.